src/components/Dropdown/generateDropdownId.ts

```diff
@@ -5,6 +5,7 @@
  * @param screenReaderUUID - the shared UUID for the {@link Dropdown} instance
  * @param index - the index of the dropdown item in question
  */
-export function generateDropdownId(screenReaderUUID: string, index: number): string {
+export function generateDropdownId(screenReaderUUID: string | undefined, index: number): string {
+  if (!screenReaderUUID) return '';
```

Comment:
can we change the other `screenReaderUUID && generateDropdownId()` usages to just `generateDropdownId()`?